content/livesync/connector/index.textile

```diff
@@ -4,11 +4,17 @@ meta_description: "The Ably Database Connector is available as a Docker containe
 product: livesync
 ---
 
-The Database Connector uses the "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html to broadcast application-specific change events from your database to frontend clients via Ably "channels":/channels.
+The Ably Database Connector (ADBC) sends updates from your database to frontend clients through Ably "channels":/channels using the "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html. Whenever you update data in your database, this system records changes in an outbox table. The Database Connector detects these changes and publishes them as messages to specified channels. Client applications using the "Models SDK":/livesync/models/models subscribe to these channels to receive updates and refreshes to their local data. 
```

Comment:
"this system" to me implies that the system does it for you automatically, but that is not the case, the developer has to record the change in the outbox table

Reply:
Also maybe remove "refreshes"?

Reply:
👍